### PR TITLE
Feature/no permission endpoints

### PR DESCRIPTION
**Pull request by JustinBBD**
# Description

Created 3 endpoitns that don;t require special permissions

1. /auth - check if you have a valid jwt
2. /organisations/create - create an organisation 
3. /organisations/get - get a list of all the orgs and spaces you are apart of

Also added the logic to check if you are the admin of a specific org, can be tested in permisssions/admin

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### CODE REVIEW

File: api/Middleware/authorizationMiddleware.js [hunk 1]
```diff
@@ -1,24 +1,59 @@
...
-    catch { return res.status(401).json({ error: 'Access denied' }); }
-  };
+    return function (req, res, next) {
+        try {
+            const query =
+                'SELECT * FROM user_space_organization_permissions where username=$1 AND  organization_name=$2 AND space_name=$3 and Permission_name=$4;';
+            const vals = [
+                req.user || '',
+                req.header('Organization') || '',
+                req.header('Space') || '',
```
Comment (ryanbasiltrickett) [inline api/Middleware/authorizationMiddleware.js:11]: Please change these to parameters and not headers

File: api/Middleware/authorizationMiddleware.js [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+            return res.status(401).json({ error: 'Access denied' });
+        }
+    };
+}
+
+export function isOrgAdmin() {
+    return function (req, res, next) {
+        try {
+            const query = 'SELECT is_user_admin($1,$2)';
+            const vals = [req.header('Organization') || '', req.user || ''];
```
Comment (ryanbasiltrickett) [inline api/Middleware/authorizationMiddleware.js:39]: Same as before, please make this a permission

File: api/Middleware/authorizationMiddleware.js [hunk 3]
```diff
@@ -1,24 +1,59 @@
...
+        } catch {
+            return res.status(401).json({ error: 'Access denied' });
+        }
+    };
+}
+
+export function isOrgAdmin() {
+    return function (req, res, next) {
+        try {
+            const query = 'SELECT is_user_admin($1,$2)';
```
Comment (ryanbasiltrickett) [inline api/Middleware/authorizationMiddleware.js:38]: Why are we using a table function over a view?

File: api/app.js [hunk 4]
```diff
@@ -33,6 +34,7 @@ app.use(express.json());
 
 app.use('/', healthCheckRouter);
```
Comment (ryanbasiltrickett) [inline api/app.js:35]: rename router to something more descriptive

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: api/Middleware/authorizationMiddleware.js (hunk 3 above)
Comment (JustinBBD) [reply api/Middleware/authorizationMiddleware.js:38]: In this case it just returns a bool, as there is only 1 organisation level role

**Review by ryanbasiltrickett — APPROVED** [review]
LGTM

**Pull request merged**